servicedirectory/snippets_test.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+import pytest
+import snippets
+from google.cloud import servicedirectory_v1beta1
+
+PROJECT_ID = environ['GOOGLE_CLOUD_PROJECT']
+LOCATION_ID = environ['GCLOUD_LOCATION']
```

Comment:
Is it required to specify a location to use this library?

We don't currently have a env var for this in CI, so we would have to add one if that is the case. 

```
Traceback (most recent call last):
  File "/workspace/servicedirectory/snippets_test.py", line 23, in <module>
    LOCATION_ID = environ['GCLOUD_LOCATION']
  File "/usr/local/lib/python3.7/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'GCLOUD_LOCATION'

Reply:
The user needs to specify the location since its part of the resource path, i.e. i specify a namespace, you need to specify the project and the location that it is in.

Where would we need to add the env var for this? For now, I've just been setting it manually in my environment (`export GCLOUD_LOCATION='us-east1'`)